### ISSUE TITLE
Sometimes i get invalid session after login when i try to execute a command and sometimes it works fine. Is this a known issue?

### ISSUE THREAD

**Issue by atulg-hf** [Caleb9/syno-photos-util#2]
Sometimes i get invalid session after login when i try to execute a command and sometimes it works fine. Is this a known issue?

**Comment by Caleb9** [comment]
No, it isn't. Does it happen shortly after you execute the `login` command?

**Comment by atulg-hf** [comment]
Yes it happens immediately after login as well. It gives following error.
  

  
Error: invalid session
  

  
And when I run the same again, it sometimes runs and sometimes gives this error. This happens about 33% of the times.
  

  
Thanks
  
Atul Gupta
  

  

  

  
  
  
  
  
>

**Comment by Caleb9** [comment]
Are you using Quick Connect? Can you take a look in the `$HOME/.syno-photos-util` if the address of your Synology looks correct?

You can also try to run the app with `RUST_LOG=debug syno-photos-util ...` and see if the output reports anything interesting, although that error is caused by the Synology API rejecting the session id, so it's unlikely we'll see anything useful.

**Comment by atulg-hf** [comment]
No, I am using it with local LAN IP only as it is within the same network and not using quick connect.
  

  

  

  
  
  
  
  
>
